tsl/src/chunk_copy.c

```diff
@@ -1302,6 +1315,13 @@ chunk_copy_cleanup(const char *operation_id)
 		}
 	}
 
+	if (!superuser() && !has_rolreplication(GetUserId()) &&
+		(ts_rel_get_owner(cc->chunk->hypertable_relid) != GetUserId()))
+		ereport(ERROR,
+				(errcode(ERRCODE_INSUFFICIENT_PRIVILEGE),
+				 (errmsg("must be superuser or replication role or hypertable owner to cleanup a "
+						 "chunk copy/move operation"))));
```

Comment:
```suggestion
				 (errmsg("must be superuser, replication role, or hypertable owner to cleanup a "
						 "chunk copy/move operation"))));
```

Reply:
Suggest adding test case for this error.

Reply:
done